README.md

```diff
@@ -70,33 +74,6 @@ To revert to previous deploy run call the `revert_deploy` playbook with a `host_
 ansible-playbook -e host_group=mep_qa playbooks/revert_deploy.yml
 ```
 
-## Overrides
```

Comment:
imo changelog makes sense in the context of releases but less sense without them, since you can just look at the list of commits. PUL has an interesting convention of [recording architecture-related decisions in markdown in the repo](https://github.com/pulibrary/princeton_ansible/tree/main/architecture-decisions), which seems sorta like our geniza decision log - maybe we could borrow that? not sure this would merit inclusion there though either.

Reply:
Yes, I'm not sure either.  A decision log or date-based change log might be useful. The commit history isn't always intelligible enough for bigger picture stuff, especially over the long term.

Reply:
we do like the [architectural decisions](https://adr.github.io/) (I personally find it liberating) in that it allows me to be wrong and to return to the decision.

Reply:
Split out into #51.